views/user.go

```diff
@@ -90,80 +92,71 @@ func (v *Views) UsersFunc(w http.ResponseWriter, r *http.Request) {
 		default:
 			valid = false
 		}
-		r.Method = "GET"
+		c.Request().Method = "GET"
+		var redirect string
 		if valid {
 			if len(search) > 0 {
 				if size > 0 {
-					http.Redirect(w, r, fmt.Sprintf("/internal/users?column=%s&direction=%s&search=%s&size=%d&page=1", column, direction, url.QueryEscape(search), size), http.StatusFound)
+					redirect = fmt.Sprintf("/internal/users?column=%s&direction=%s&search=%s&size=%d&page=1", column, direction, url.QueryEscape(search), size)
 				} else {
-					http.Redirect(w, r, fmt.Sprintf("/internal/users?column=%s&direction=%s&search=%s", column, direction, url.QueryEscape(search)), http.StatusFound)
+					redirect = fmt.Sprintf("/internal/users?column=%s&direction=%s&search=%s", column, direction, url.QueryEscape(search))
 				}
 			} else {
```

Comment:
DIdn't I comment on refactoring this to use a query builder in #10?

Reply:
Yes and that is still on it's way, it is done a bit later